pavlolucik/docker/04graduationWorkBuildJar/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu
+LABEL maintainer="Pavlo Lucik pavel.lucik@gmail.com"
+RUN apt-get update
+RUN apt-get install -y git \
+ openjdk-8-jdk \
+ maven
+RUN git clone https://github.com/pavel-lucik/gw-spring-boot.git
```

Comment:
Do not put each shell command into a dedicated RUN command. It is redundant and creates many unnecessary layers.